webpack.config.js

```diff
@@ -65,11 +65,15 @@ const webpackConfig = {
 			{
 				include: /node_modules[\/\\]tinymce/,
 				loader: 'imports?this=>window',
+			},
+			{
+				test: /\.(ml|re)$/,
```

Comment:
You'll probably also want to add the `.mli` and `.rei` extensions to load any module interface files. Those are like C header files and are super useful in OCaml.